### ISSUE TITLE
ASM load not correct

### ISSUE THREAD

**Issue by Gilph** [BobPalmer/MKS#1043]
Hi,
Having an intermittent issue with the ASM module.  I have two bays assigned to H2O(Hyd), but the display screen shows the load % for only one bay (145,91).  I sometimes see the correct load number (2x) but there is no definite way to reproduce  it.  Please see attached screenshot.  Hydrates level are sufficient for both bays.
![screenshot19](https://cloud.githubusercontent.com/assets/23642289/21075758/eead2b22-bee7-11e6-9ffd-096fb66c1707.png)

**Comment by Gilph** [comment]
Found a way to reproduce. I go to Tracking, select a vessel close by, cycle through the vessels with the bracket keys to this vessel, it will report incorrectly. If I go to Tracking and select this vessel directly, it will report correctly.  It is not a display issue only, when the incorrect status is shown there is half as much water being produced.

![screenshot27](https://cloud.githubusercontent.com/assets/23642289/21080901/c58e975c-bf88-11e6-84d8-cf54e88307e8.png)

**Comment by Gilph** [comment]
Logs from my career save...one with the issue and one without.  USI Only save in this location:

https://1drv.ms/f/s!AigvIEvgjetqvjxo_ayTvjaBNLkP

Thanks

**Comment by Gilph** [comment]
Save directory copied to above shared link.  This is from the new versions released, still with issue. To reproduce:

1. From KSC, go to Tracking
2. Select MinBase2PDU
3. hit the bracket keys to cycle through the vessels to MinBase2Supplies. One AM module will have Agroponics and Cultivate(S). The single Cultivate(S) will be about 159%. The other AM will have two Cultivate(S), and will also be at 159%.  This should be double the value because two bays are enabled.
4. hit the bracket keys to select MinBase2Water. The two ASMs are configured the same: one Fertilizer(G) and two H2O(Hyd). They will both be at around 166%, but the H2O should be double the load because it has two bays.
5. Go to KSC and hit Tracking
6. Select MinBase2Supplies
7. Check the load of the AM with two Cultivate(S).  It is correctly set as 318%, which is different from the earlier step.
8. hit the bracket keys to select MinBase2Water. The error will still be there.
9. Go to KSC and hit Tracking
10. Select MinBase2Water
11. Check each ASM. The load is now correct (about 333%)
12. hit the bracket keys to select MinBase2Supplies. Check the load of the AM with two Cultivate(S). It is now reduced back to 159%.
13. 

In summary, anytime a part is configured to have two or more bays produce the same resource, it will only produce it at a single bay rate, _unless it is selected directly from Tracking_.
